src/NuGet.Services.BasicSearch/Web.config

```diff
@@ -4,6 +4,15 @@
   http://go.microsoft.com/fwlink/?LinkId=169433
   -->
 <configuration>
+  <system.diagnostics>
+    <trace>
+      <listeners>
+        <add type="Microsoft.WindowsAzure.Diagnostics.DiagnosticMonitorTraceListener, Microsoft.WindowsAzure.Diagnostics, Version=2.8.0.0, Culture=neutral, PublicKeyToken=31bf3856ad364e35" name="AzureDiagnostics">
```

Comment:
This one was added by the NuGet package upgrade. Seems Azure diagnostics changed configuration settings again? Happy to remove it if not needed.